### PR TITLE
feat: add codeowners file

### PR DESCRIPTION
**Pull request by janderssonse**
Adds codeowners file and recommendations, with reps from Digg and asom. For now, @danneleaf @Ayko1595.

## Checklist

- [x] My contributions and commit messages follows the style guidelines of this project
- [x] I have made corresponding changes to the documentation
- [x] New and existing unit tests pass locally with my changes
- [x] The Pull Request has an informative and human-readable title
- [x] Changes are limited to a single goal (avoid scope creep)
- [x] Code can be automatically merged (no conflicts)
- [x] I confirm that I have read any Contribution guidelines (CONTRIBUTING)
- [x] I confirm that I wrote and/or have the right to submit the contents of my PR, by agreeing to the _Developer Certificate of Origin_, by adding a 'sign-off' to my commits

### CODE REVIEW

**Comment by janderssonse** [comment]
As spoke about today in meeting, another pr with the rest of the suggested improvements will follow when i get some time, this is only the CODEOWNERS part. Don't merge, just approve if so please:)

**Comment by Ayko1595** [comment]
This looks good! We discussed adding more clarification about what it means to be a "codeowner". Should this be added in this PR aswell?

**Comment by janderssonse** [comment]
Thanks!!:) nay, I think we should save that to an near-in-time governance-topic meeting!

**Review by Ayko1595 — APPROVED** [review]

**Pull request merged**
